design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -224,6 +224,56 @@ pod.
     - Mount points
     - Downward API
 
+##### Compute Container Resource Overhead
+
+For some plugins, an additional resource consumption can be expected from the virt-launcher pod compute container.
+For example, there could be need to execute an additional binary in the compute container.
+Since this binary has its own CPU and memory limits, they should be somehow accounted for.
```

Comment:
I think, we should speak of a specific functionality and not about arbitrary binaries.
virt-launcher should be well defined. 
If we know what functionality we're enabling that requires additional overhead, kubevirt needs to know the resources it requires. Similar to other aspects accounted for in `GetMemoryOverhead`.

I would say that if we want an API for this it should be based on a functionality, not a binding. 

 Each resource consumption of it.